mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -87,11 +92,18 @@ class UcModelRegistryStore(BaseRestStore):
     """
     Client for a remote model registry server accessed via REST API calls
 
-    :param get_host_creds: Method to be invoked prior to every REST request to get the
-      :py:class:`mlflow.rest_utils.MlflowHostCreds` for the request. Note that this
-      is a function so that we can obtain fresh credentials in the case of expiry.
+    :param registry_uri: URI with scheme 'databricks-uc'
+    :param tracking_uri: URI of the Databricks MLflow tracking server from which to fetch run info and download run
```

Comment:
The UC registry rest store needs both registry and tracking URIs, for the reason described here - it needs to download run artifacts from the tracking location in order to subsequently register them to the registry storage location.

For context: we're moving model version creation/file upload into the client (vs in the server) with the UC model registry, similar to how we log model artifacts client-side in MLflow tracking. This helps unblock better scale and registration of large models, etc